package.json

```diff
@@ -1,29 +1,31 @@
 {
   "name": "hyperverse",
-  "version": "1.0.0",
+  "version": "0.0.0",
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "lint": "tslint -c tslint.json 'src/**/*.ts'",
+    "release": "standard-version",
+    "build": "tsc -p ."
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "eslint": "^5.16.0",
     "injection-js": "^2.2.1",
-    "libp2p": "^0.25.4",
+    "libp2p": "^0.25.5",
     "libp2p-secio": "^0.11.1",
     "libp2p-spdy": "^0.13.3",
     "libp2p-tcp": "^0.13.0",
     "peer-info": "^0.15.1",
     "pull-pushable": "^2.2.0",
-    "pull-stream": "^3.6.12",
+    "pull-stream": "^3.6.13",
     "reflect-metadata": "^0.1.13"
   },
   "devDependencies": {
-    "@types/node": "^12.0.8",
-    "source-map-support": "^0.5.12",
-    "tslint": "^5.17.0",
-    "typescript": "^3.5.2"
+    "@types/node": "^12.6.9",
```

Comment:
I don't think this applies. It should not be node specific for now

Reply:
Ok, I'll revert this later!